docs/Getting-Started.md

```diff
@@ -0,0 +1,178 @@
+To get started hacking with Tux, let's see what some parts of a to-do component might look like inside a To-Do Application built with Tux.
+
+To begin with we'll require only the modules we need to get our component up and running:
```

Comment:
Need to set up `browserify` and the file structure first. Check out the README because it has guidance on this.